File: URI/python/1045.py
lados = [float(x) for x in input().split()]
lados.sort()
lados = lados[::-1]
a, b, c = list(lados)

if a >= b + c:
    print('NAO FORMA TRIANGULO')
else:
    if a ** 2 == b ** 2 + c ** 2:
        print('TRIANGULO RETANGULO')
    if a ** 2 > b ** 2 + c ** 2:
        print('TRIANGULO OBTUSANGULO')
    if a ** 2 < b ** 2 + c ** 2:
        print('TRIANGULO ACUTANGULO')
    if a == b == c:
        print('TRIANGULO EQUILATERO')
    if a == b != c or a != b == c or a == c != b: 
        print('TRIANGULO ISOSCELES')

File: uri/c/1045.c
#include <stdio.h>
#include <math.h>

int main() {

    double A, B, C, maior, meio, menor;
    scanf("%lf %lf %lf", &A, &B, &C);

    maior = A;
    meio = B;
    menor = C;

	if (B >= A && B >= C){
		maior = B;
		meio = A;
		menor = C;
	} else {
		maior = C;
		meio = A;
		menor = B;
	}

    A = maior;
    B = meio;
    C = menor;

    if (A >= (B + C)) {
        printf("NAO FORMA TRIANGULO\n");
    } else {

        if (pow(A, 2) == (pow(B,2) + pow(C,2))) {
            printf("TRIANGULO RETANGULO\n");
        } else if (pow(A, 2) > (pow(B,2) + pow(C,2))) {
            printf("TRIANGULO OBTUSANGULO\n");
        } else if (pow(A, 2) < (pow(B,2) + pow(C,2))) {
            printf("TRIANGULO ACUTANGULO\n");
        }

        if (A == B && B == C) {
            printf("TRIANGULO EQUILATERO\n");
        } else if (A == B || A == C || B == C) {
            printf("TRIANGULO ISOSCELES\n");
        }


    }

    return 0;

}